tests/parsers/test.fluent.js

```diff
@@ -65,29 +38,26 @@ key69
     parser.parse();
 
     expect(parser.isValid).toEqual(true);
-    expect(parser.parsedData).toEqual({
-      key67: {
-        attrs: {
-          accesskey: 'Y',
-          label: 'Sign In To &syncBrand.shortName.label;…',
-        },
-        val: undefined,
-      },
-      key68: {
-        attrs: {
-          accesskey: 'S',
-          label: 'Sync Now',
-        },
-        val: undefined,
-      },
-      key69: {
-        attrs: {
-          accesskey: 'R',
-          label: 'Reconnect to &syncBrand.shortName.label;…',
-        },
-        val: undefined,
-      },
-    });
+    expect(parser.parsedData.key67.attributes[0].value.elements[0].value).toEqual(
```

Comment:
These tests read foreign to me without context but I have no idea how fluent works... I'm gonna trust you on them 😄